### PR TITLE
Add automation to release proccess

### PR DESCRIPTION
**Pull request by yanirq**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,17 +4,39 @@ go: "1.10"
...
 - docker
 
-install: true
+install:
+   - make manifests
+
 script: make container-build 
 
+before_deploy:
+- docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io
```
Comment (ksimon1) [inline .travis.yml]: you should rather use this syntax:
 echo "$DOCKER_PASS" | docker login -u="$DOCKER_USER" quay.io --password-stdin
Usage of -p is insecure in cli.

File: .travis.yml [hunk 2]
```diff
@@ -4,17 +4,39 @@ go: "1.10"
...
     tags: true
+# Release assests
+- provider: releases
+  branches:
+    only: /v[0-9]+\.[0-9]+\.[0-9]+(-alpha\.[0-9]+){0,1}/
+  on:
+    tags: true
+    repo: kubevirt/node-maintenance-operator
+  api_key:
+    secure: vu7hEF10GBxbVRbWZ6wltZTNDfMZXUzRCsmtHGwmQ7z2kpMlGJkuGpIp7+eTkgHlFUPA2bmjSTkuO8nyVWLWgfx/Pf/mIZYUSYdLKg+N+KDIEqF0dk9GLYE6oXpF9bywgW1LollUygTuaHe/nvQVHElfdFSs4PAQabGnCDarsyEJPLPNJAUDmTtfm2wiSzVOWpvmIXucQKWRz5bMi1nf0z8+0bPUo7PkaTX2CvbcCkb8EXZ99+x4wjp/AqRGwNFpQUbtRn5YaRuZUC3wjT0Oar+/INHCMxIW4dzHsOEQhQgRx/UUW+0S9LZ7MGe5rLZHl0XT4Cxn7LuCEi3TzclnQJwyt2VNFususIxKPzn5T1cV67EG50k0JR9QmNsOIQTLiZkCT17ne3/cs5CLn5WBBFMsbf7bK8B2ne7ZXaQdZE9m0HBh2ArEnhjBaM/zKSwIX7iqFhidE55QZGuQ5pwRru44tFFDllkC3lsVCOOYjbEGBftqV7HmUOhfr//l7CR3s4LM7IywVKq3D2nZQ4zwSkUxzicVTrS4w4PQtz4jim6cIvvs1AZI6wj1iqhrJcFVqA8v7P63XEcBaHT+1CNJ/kkmOk2s8M06GEQ1sHCqBNk4AJY1hvVuGcV9Z+lhS19KadJZAXsqR9v1+0bRw1OjteQRewIcbROVzQ2k3Zi8Swk=
```
Comment (MarSik) [inline .travis.yml]: I prefer storing the key in environment variable, it allows easier updates and repo cloning

**Comment by gbenhaim** [comment]
I've created the jobs, please set up webhooks and permissions to the bot:
```
Hook for PR events:
 
content-type: application/x-www-form-urlencoded
hook url: https://kubevirt-master-stdci-production.apps.ovirt.org/ghprbhook/

check boxes to mark: Pull requests, Issue comments
 
Hook for Push events:

 
content-type: application/json

hook url: https://kubevirt-master-stdci-production.apps.ovirt.org/generic-webhook-trigger/invoke

check boxes to mark: Just the push event.
 
This is a manual about how to set them (please make sure to use the urls I've posted above). In the manual search for "Adding a GitHub hook for handling PR events"- https://ovirt-infra-docs.readthedocs.io/en/latest/CI/Using_STDCI_with_GitHub/index.html#using-ovirt-standard-ci-with-github

 

You will also need to give write permissions on the repo to "ovirt-infra" bot, so it would be able to write the status of the jobs in the MR.

 

The job was created - https://kubevirt-master-stdci-production.apps.ovirt.org/job/kubevirt_node-maintenance-operator_standard-check-pr/

Patch - https://gerrit.ovirt.org/#/c/99818/
```

File: .travis.yml (hunk 2 above)
Comment (yanirq) [reply .travis.yml]: This is a GitHub oauth token that was automatically created and encrypted under the correct scopes (using travis setup releases).
I just followed the convention here.

**Comment by yanirq** [comment]
ci test please

File: .travis.yml (hunk 2 above)
Comment (MarSik) [reply .travis.yml]: You can pass the same info using Travis environment variables. I prefer to not put the key into the  file directly myself. Both approaches work though.

**Pull request merged**